### PR TITLE
[GIE IR] Add docker files to build images of GIE (using exp store)

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
1. add docker files under `k8s/standalone` to build gie with exp store.
2. add `start_rpc_server_k8s` to start executor processes in single or distributed env.

## Related issue number

Fixes

### CODE REVIEW

File: interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+use std::net::SocketAddr;
```
Comment (longbinlai) [inline interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs:1]: Copyright.

File: interactive_engine/compiler/set_properties.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (longbinlai) [inline interactive_engine/compiler/set_properties.sh:1]: Copyright

File: interactive_engine/executor/engine/pegasus/server/src/cluster/mod.rs [hunk 3]
```diff
@@ -1 +1,2 @@
+pub mod k8s;
```
Comment (longbinlai) [inline interactive_engine/executor/engine/pegasus/server/src/cluster/mod.rs:1]: copyright

File: interactive_engine/compiler/set_properties.sh (hunk 2 above)
Comment (shirly121) [reply interactive_engine/compiler/set_properties.sh:1]: done

File: interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs (hunk 1 above)
Comment (shirly121) [reply interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs:16]: done

Comment (longbinlai) [reply interactive_engine/executor/engine/pegasus/server/src/cluster/k8s.rs:16]: This file called `k8s.rs`. Donot think is appropriate.

File: interactive_engine/executor/ir/integrated/src/bin/start_rpc_server_k8s.rs [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    pegasus_common::logs::init_log();
+    let id = get_id();
+    let servers_size = get_servers_size();
+    let dns_name = get_dns_name();
+    let engine_port = get_engine_port();
+    let rpc_port = get_rpc_port();
+    let server_config_string = generate_server_config_string(id, servers_size, &dns_name, engine_port);
```
Comment (longbinlai) [inline interactive_engine/executor/ir/integrated/src/bin/start_rpc_server_k8s.rs:29]: Can we implement a structure with some generic type to handle start_rpc_server standalone and with k8s? Can also do so for k8s.rs and standalone.rs.

**Review by longbinlai — APPROVED** [review]

**Pull request merged**